starter-code/app.js

```diff
@@ -52,7 +36,59 @@ app.locals.title = 'Express - Generated with IronGenerator';
 
 
 const index = require('./routes/index');
+const postsRoute = require('./routes/posts');
+
+app.use(
+  session({
+    secret: "wow much secret, very secret", //encrypts cookie (so it hashes)
+    cookie: {
+      maxAge: 24 * 60 * 60 // 1 day
+    }, // options for cookie storage
+    resave: false, //don't save session if unmodified
+    saveUninitialized: false, // don't create session until something stored
+    store: new MongoStore({
+      mongooseConnection: mongoose.connection,
+      ttl: 24 * 60 * 60 // 1 day
+    })
+  })
+);
+
 app.use('/', index);
+app.use("/posts", postsRoute);
+
+// Express View engine setup
+
+app.use(require('node-sass-middleware')({
+  src: path.join(__dirname, 'public'),
+  dest: path.join(__dirname, 'public'),
+  sourceMap: true
+}));
+
+
+
+mongoose
```

Comment:
I would probably keep this connection in different file. App.js should be as slim as possible